app/controllers/favorites_controller.rb

```diff
@@ -1,5 +1,12 @@
 class FavoritesController < ApplicationController
   def index
+    apps = PetApplication.all
+    @pet_apps = []
+    if !apps.nil?
+      apps.each do |app|
+        @pet_apps << Pet.find(app[:pet_id])
+      end
+    end
```

Comment:
Good work on this method! I think this one and the other one I wrote will eventually live in the model but this works.